### ISSUE TITLE
cannot change font or font size

### ISSUE THREAD

**Issue by drammock** [spyder-ide/spyder#3113]
## Description of your problem
possibly related to #3007 

**What steps will reproduce the problem?**

1. Start spyder
2. Open preferences
3. Change typeface or font size settings

**What is the expected output?**
font in editor, console, etc should change

**What do you see instead?**
typeface and font size do not change.  

**Please provide any additional information below**
After closing and re-opening preferences, the values in the typeface and font size fields revert to displaying whatever they are set to in the `spyder.ini` file.

## Versions and main components
```
[~] $ python --version
Python 2.7.6
[~] $ lsb_release -d
Description:	Ubuntu 14.04.4 LTS
[~/B/spyder] $ git rev-parse HEAD
021874d0aa7ead3f4892ac0a4f16378163acaba6
```

## Dependencies
jedi >=0.8.1    :  0.9.0 (OK)
matplotlib >=1.0:  1.5.1+1643.g0423430 (OK)
nbconvert >=4.0 :  4.2.0 (OK)
numpy >=1.7     :  1.12.0.dev0+2af06c8 (OK)
pandas >=0.13.1 :  0.18.0+104.g1c8816f (OK)
pep8 >=0.6      :  1.7.0 (OK)
psutil >=0.3    :  1.2.1 (OK)
pyflakes >=0.5.0:  1.1.0 (OK)
pygments >=1.6  :  2.1.3 (OK)
pylint >=0.25   :  1.5.5 (OK)
qtconsole >=4.0 :  4.2.1 (OK)
rope >=0.9.4    :  0.10.3 (OK)
sphinx >=0.6.6  :  1.5a0 (OK)
sympy >=0.7.3   :  None (NOK)

**Comment by ccordoba12** [comment]
This was fixed in commit 37bfbe080d3e2ac66a3536c27ef13ba331c3a55a.

Sorry, I didn't notice your issue but I noticed this problem myself, so I fixed it directly.

**Comment by drammock** [comment]
awesome thanks @ccordoba12
